src/FromFile.jl

```diff
@@ -12,6 +12,38 @@ function __init__()
     @require Revise="295af30f-e4ad-537b-8983-00126c2a3abe" track(mod, path) = Revise.track(mod, path)
 end
 
+"""
+    @from <path/or/url/to/file>
+    @from <path/or/url/to/file> using my_module
+    @from <path/or/url/to/file> import my_object
+    @from <path/or/url/to/file> import my_module: my_object
+
+Import your object from another Julia script by treating the
+script as a module.
+
+# Examples
+
+In `basic.jl` it contains
+
+```julia
+module A
+    foo() = print("hello")
+end
+```
+
+and `A` can be imported via
+
+```julia
+@from "basic.jl" import A
+```
+
+or we can import `foo` that is inside `A`
+
+```julia
+@from "basic.jl" using A: foo
+@from "basic.jl" import A: foo
+```
+"""
```

Comment:
This doesn't seem super readable to me.

I'd mention URLs separately at the end, as they're a more niche use case, so better to introduce the concept of `@from` without that first.

I'd avoid using `module` in the example as that isn't necessary in a minimal use-case.

I'd switch the order of the example: show the  file containing the `@from` macro before the one being imported; this shows the important `@from` up-front.

Reply:
> I'd mention URLs separately at the end, as they're a more niche use case, so better to introduce the concept of @from without that first.

ah right, good point, I wrote this before #42, let's wait #42 get merged, then I'll put some more description in this PR.